### PR TITLE
Ballchasy - Bug fix & Progress

### PR DESCRIPTION
**Pull request by BenjaminHunt**
# bcManager
- updates score reporting to use `asyncio.to_thread`
- adds an `info.json` to the cog

This is required an update to the bot's python version to be updated to a minimum of `python 3.9.0`

### CODE REVIEW

**Review by blister — APPROVED** [review]

**Comment by BenjaminHunt** [comment]
image
One comment on this report -- The "[View Group](https://ballchasing.com/group/match-day-12-5gxtpehjlo)" link for the tier does not appear in this example because of the 2 matches present, one match was previously reported, while replays were never found for the other, so the replay location traversal never occurred for the tier on this execution, and no tier group was populated at run time. 

I believe this to be a minor issue worth noting, but not addressing.

If anyone believes otherwise, it will involve either refactoring the match cog saved data (injecting the group link into the tiers) or by executing the traversal before the replay search, which I believe would be a poor choice considering rate limiting limits.

**Review by xentrick — APPROVED** [review]

File: bcManager/bcManager.py [hunk 1]
```diff
@@ -185,10 +185,15 @@ async def reportMatches(self, ctx, match_day: int=None):
         
         # update status message
         await processing_status_msg.edit(embed=self.get_bc_match_day_status_report(match_day, bc_report_summary_json, emoji_url=guild_emoji_url, complete=True))
-                
-        
         # TODO: enable:
         # await self.process_missing_replays(ctx.guild, all_missing_replays)
+             
+    @commands.guild_only()
```
Comment (adam-stover) [inline bcManager/bcManager.py:191]: nit do we want this in prod or nah?

File: bcManager/info.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "author" : ["RSC Development Committee"],
+    "description" : "A ballchasing integration with Rocket Soccar Confederation (RSC) for league match replay management and score reporting.",
+    "short" : "RSC ballchasing integration.",
+    "requirements" : ["python-ballchasing"],
+    "permissions" : [],
+    "tags" : ["RSC"],
+    "min_python_version": [3, 9, 0]
+}
```
Comment (adam-stover) [inline bcManager/info.json:9]: nit newline EOF

File: match/misc/example_settings.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                },
+                "LobbyHashes": {
+                    "1": [-8600427675859063065, 7192519824993810708],
+                    "2": [4301634076316122128, 6282479947756218303]
+                },
+                "MatchDay": "4"
+            }
+        }
+    }
+}
```
Comment (adam-stover) [inline match/misc/example_settings.json:44]: nit newline EOF

**Review by adam-stover — APPROVED** [review]

**Pull request merged**